examples/src/generic.rs

```diff
@@ -0,0 +1,102 @@
+use yaserde::ser::to_string;
+use yaserde::*;
+use yaserde::de::from_str;
+
+#[derive(YaSerialize, YaDeserialize, Debug, Default, Clone, Eq, PartialEq)]
+pub struct Header {}
+
+#[derive(YaSerialize, YaDeserialize, Debug, Default, Clone, Eq, PartialEq)]
+#[yaserde(
+  rename = "Envelope",
+  namespace = "s: http://schemas.xmlsoap.org/soap/envelope/",
+  prefix = "s"
+)]
+pub struct SoapEnvelope<BODY>
```

Comment:
It's great to have a functional test here.
But it can be great to add unit test to have safe next developments.